client/src/components/Room.jsx

```diff
@@ -33,6 +33,17 @@ class Room extends React.Component {
   componentDidMount() {
     connectionManager.setup(this.props.params.roomId);
     connectionManager.onStatusChange(this.updateConnection);
+  // this will update uniue user instruments with those made in the same session.
+    $.get("/userLoggedInToMakeInst", (resp, err) => {
+      console.log('Resp to RDM:', resp);
```

Comment:
Initially it was to make sure that folks couldn't access /makeinstruments without having being logged in. Can rename to something like "persist".